templates/pages/core/get-started/index.md

```diff
@@ -1,24 +1,26 @@
 ---
 title: Get started
-description: Your first steps, installing Ubuntu Core on a supported board or in a virtual machine
+description: Your Ubuntu Core first steps, installing Ubuntu Core on your board or in a Virtual Machine.
 ---
 
 # Get started
 
-Ubuntu Core is supported on a variety of chipsets, boards, SOCs and virtual machines, including Raspberry Pi 2 and 3, Qualcomm DragonBoard 410c, Intel NUC, Intel Joule, Samsung Artik and KVM. This section will guide you through the steps of installing Ubuntu Core on these platforms.
+Ubuntu Core is supported on a variety of chipsets, boards, SOCs and virtual machines, including Raspberry Pi 2 and 3, Qualcomm DragonBoard 410c, Intel NUC, Intel Joule, Samsung Artik and KVM.
 
-## [1. Create installation medias ›](/core/get-started/installation-medias)
+This section will guide you through the steps of installing Ubuntu Core on these platforms.
```

Comment:
Maybe change 'This section' to 'We will..' or something more generic as could be confused with intro'ing the Create installation media section.